util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -498,6 +498,7 @@ module chip_${top["name"]}_${target["name"]} #(
   // pwrmgr interface
   pwrmgr_pkg::pwr_ast_req_t base_ast_pwr;
   pwrmgr_pkg::pwr_ast_rsp_t ast_base_pwr;
+  pwrmgr_pkg::pwr_boot_status_t pwrmgr_boot_status;
```

Comment:
Guard that with Darjeeling so this does not get defined for Eargrey

Reply:
done